src/test/java/assertions/PersonTest.java

```diff
@@ -43,12 +43,22 @@ public void getAgeRetunObAlterRichtigRechnet() {
 
     // TODO verbessern. Hinweis: Repeatable (wiederholbar) zu jeder Zeit.
     @Test
-    public void getAgeReturns10YearsIf2006() throws Exception {
-        Person p = new Person("", "", LocalDate.of(2006, 1, 1));
+    public void getAgeReturns10YearsIf2006() throws Exception{
+        Person p = new Person("", "", LocalDate.of(2006, 1, 1));}
 
+        public void getAgeGibtImmer10JahreZurueck(){
```

Comment:
Bei dieser Methode fehlt die @Test Annotation. Das heisst diese Methode wird nicht ausgeführt, wenn du z.B. sagst "Run All Tests in PersonTest" o.ä.